QUICKSTART.md

```diff
@@ -0,0 +1,354 @@
+    resources: [ "apibindings" ]
+    resourceNames: [ "bind-kube" ]
+  upsync:
+  - apiGroup: "group1.test"
+    resources: ["sprockets", "flanges"]
+    namespaces: ["orbital"]
+    names: ["george", "cosmo"]
+  - apiGroup: "group2.test"
+    resources: ["cogs"]
+    names: ["william"]
+EOF
+```
+
+Put the prescription of the HTTP server workload into the WMW. Note the namespace label matches the label in the namespaceSelector for the EdgePlacement (`edge-placement-c`) object created above. 
+
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: commonstuff
+  labels: {common: "si"}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: commonstuff
+  name: httpd-htdocs
+data:
+  index.html: |
+    <!DOCTYPE html>
+    <html>
+      <body>
+        This is a common web site.
+      </body>
+    </html>
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: commonstuff
+  name: commond
+spec:
+  selector: {matchLabels: {app: common} }
+  template:
+    metadata:
+      labels: {app: common}
+    spec:
+      containers:
+      - name: httpd
+        image: library/httpd:2.4
+        ports:
+        - name: http
+          containerPort: 80
+          hostPort: 8081
+          protocol: TCP
+        volumeMounts:
+        - name: htdocs
+          readOnly: true
+          mountPath: /usr/local/apache2/htdocs
+      volumes:
+      - name: htdocs
+        configMap:
+          name: httpd-htdocs
+          optional: false
+EOF
+```
+
+Now, let's check that the deployment was created in the `florin` edge cluster:
+
+```shell
+kubectl --context kind-florin get deployments -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE     NAME      READY   UP-TO-DATE   AVAILABLE   AGE
+commonstuff   commond   0/0     0            0           6m44s
+```
+
+Also, let's check that the deployment was created in the `guilder` edge cluster:
+
+```shell
+kubectl --context kind-guilder get deployments -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE     NAME      READY   UP-TO-DATE   AVAILABLE   AGE
+commonstuff   commond   0/0     0            0           6m44s
+```
+
+Lastly, let's check that the workload is working in both clusters:
+
+For `florin`:
+
+```console
+$ curl http://localhost:8081
+<!DOCTYPE html>
+<html>
+  <body>
+    This is a common web site.
+  </body>
+</html>
+```
+
+For `guilder`:
+
+```console
+$ curl http://localhost:8082
+<!DOCTYPE html>
+<html>
+  <body>
+    This is a common web site.
+  </body>
+</html>
+```
+
+Congratulations, you’ve just deployed a workload to two edge clusters using kubestellar! To learn more about kubestellar please visit our [User Guide](<place-holder>)
+
+## 3. Teardown the environment
+
+To remove the example usage, delete the IMW and WMW and kind clusters run the following command:
+
+```shell
+kubestellar remove
+```
+
+Stop and uninstall KubeStellar and kcp with the following commands:
```

Comment:
We should explicitly say that this is stopping and uninstalling kcp too.

We should explicitly say that stopping the controllers is bundled with uninstalling here only for convenience and in general can be done without uninstalling anything.

Reply:
agreed - Braulio to update and then resolve comment

Reply:
line does state that kcp and kubestellar are stopping and uninstalling - resolving.